src/wmtk/attribute/MeshAttributes.cpp

```diff
@@ -100,6 +100,12 @@ void MeshAttributes<T>::set(const AttributeHandle& handle, std::vector<T> val)
     attr.set(std::move(val));
 }
 
+template <typename T>
```

Comment:
this size should be always equal to m_size, perhaps this naming is better but we should discuss some standard for these names